templates/index.html

```diff
@@ -33,6 +33,35 @@
   </header>
 
   <div id="builds">
+
+    <!-- B2GDroid promo -->
+    <!--  Spotlight Activity Tag: Mozilla (6247) | Mozilla B2G Activity Tag 2 (52162) | Mozilla Spots (4669) | Expected URL:  -->
+    <script>
+      function ftGoalTag52162() {
+        var ftRand = Math.random() + "";
+        var num = ftRand * 1000000000000000000;
+        var ftGoalTagPix52162 = new Image();
+        ftGoalTagPix52162.src = "http://servedby.flashtalking.com/spot/8/6247;52162;4669/?spotName=Mozilla_B2G_Activity_Tag_2&cachebuster="+num;
+        ftGoalTagPix52162.onload = ftLoaded52162;
+      }
+      function ftLoaded52162() {
+        top.location.href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk";
+      }
+    </script>
+    <div id="Firefox OS" class="build-group">
+      <h3>Firefox OS</h3>
+      <ul>
+        <li class="b2gdroid first">
+          <a href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk" onClick="ftGoalTag52162();return false;">
```

Comment:
If you want to attach functions to elements, please use event listeners in JS, not `onClick`.

Though here it looks like you're using JS to capture a click and then redirect the browser to the original link anyway, so I have no idea why the JS.